app/components/Common/InfoToolTip.tsx

```diff
@@ -1,16 +1,27 @@
 import { InformationCircleIcon } from '@heroicons/react/solid'
+import React from 'react'
 
 /**
  * This ToolTip component will position itself above it's first parent with a position 'absolute' or 'relative' css property.
  * At this moment in time it's max width will be the same as that same parent.
  */
-export const ToolTip = ({ data }: { data: string }) => (
+export const ToolTip = ({
```

Comment:
One thing you can do to make it less grumpy with Typescript,

is import `FC` then have that as the function return type.

In this case,

```
...
import {FC} from 'react';
...

type TooltipProps = {
  data: string
  parseTags?: boolean
}
export const ToolTip: FC<TooltipProps> = ({date, parseTags}) => (
<>etc</>
);

```

Reply:
Fyi it is now recommended to explicitly type children with ReactNode

https://beta.reactjs.org/blog/2022/03/08/react-18-upgrade-guide#updates-to-typescript-definitions

Reply:
That's good to know, I've always used the prop wrapper `PropsWithChildren` because I am a lazy-bones.

<img width="1130" alt="image" src="https://user-images.githubusercontent.com/924120/214422101-f644ecf3-5287-46cf-b05b-a64dc12af6f5.png">